client/assets/global.scss

```diff
@@ -83,10 +83,24 @@ body {
 
 /* Custom utility classes */
 
+.absolute-cover {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
 .pointer {
   cursor: pointer;
 }
 
+.pointer-events-none {
```

Comment:
The `pointer-events-none` and `all-pointer-events` classes should match, and also might benefit from being in the same location.